geometry_msgs/PoseStamped[] goals
---
bool success
